cider/exec/plan/parser/SubstraitToAnalyzerExpr.cpp

```diff
@@ -37,7 +37,7 @@ bool getExprUpdatable(std::unordered_map<std::shared_ptr<Analyzer::Expr>, bool>
 }
 
 bool isStringFunction(std::string function_name) {
-  std::unordered_set<std::string> supportedStrFunctionSet{"substring"};
+  std::unordered_set<std::string> supportedStrFunctionSet{"substring", "lower", "upper"};
```

Comment:
please define the input argument as `const std::string& name`
and cause the supportedStrFunctionSet only has three names, so if we want make this function more fast, codes can be written:
```c++
std::string_view names[] = {"substring", "lower", "upper"};
return std::find(std::cbegin(names), std::cend(names), name) != std::cend(names);
```
or
```c++
std::string_view names[] = {"substring", "lower", "upper"};
for (size_t i = 0; i < sizeof(names); ++i) {
  if (names[i] == name) {
    return true;
  }
return false;
```